### DISCUSSION TITLE
Biblioteca de ícones do POUI

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1777] [Tire suas dúvidas] [answered]
Anteriormente, no site do POUI, existia um guia com uma biblioteca de ícones que poderiam ser usados em nossas aplicações.

Mas, depois da última atualização do site, não estou conseguindo encontrar este guia. Anteriormente ele ficava no endereço: https://po-ui.io/guides/icons

O endereço foi removido? Existe alguma outra documentação que informe os ícones padrões que podem ser utilizados?

**Comment by felipepetuco** [comment, marked as answer]
Boa tarde página do guide icons foi restabelecida.
Atenciosamente,
